Dockerfile

```diff
@@ -0,0 +1,6 @@
+# syntax=docker:dockerfile:1
+FROM python:3.10-slim@sha256:2bac43769ace90ebd3ad83e5392295e25dfc58e58543d3ab326c3330b505283d
+WORKDIR /usr/app
+COPY . .
```

Comment:
If you only need two files and not your entire repository then it is better to `COPY` them in individually than to bring along everything.